main.tf

```diff
@@ -98,14 +98,14 @@ resource "aws_subnet" "lambda" {
   vpc_id                  = aws_vpc.default.id
 
   tags = merge(
-    var.tags, { "Name" = "lambda-${var.name}-${local.az_ids[count.index]}" }
```

Comment:
if you move public and private to behind the name, then lambda should be too I think

Reply:
fixed, good catch.